code/src/java/pcgen/gui2/coreview/CoreViewFrame.java

```diff
@@ -240,8 +238,8 @@ public void setPerspective(CorePerspective corePerspective)
 		@Override
 		public ListFacade<? extends TreeView<CoreViewNodeFacade>> getTreeViews()
 		{
-			DefaultListFacade<TreeView<CoreViewNodeFacade>> views =
-					new DefaultListFacade<TreeView<CoreViewNodeFacade>>();
+			DefaultListFacade<TreeView<CoreViewNodeFacade>> views
+					= new DefaultListFacade<TreeView<CoreViewNodeFacade>>();
```

Comment:
I use Netbeans for my pcgen work. I've adjusted its formatting settings to adhere as closely as possible the pcgen formatting standards. That said, I've looked through the Eclipse code format xml files and I cannot find (it is possible I just missed it) any settings regarding wrapping the line after the assignment operator. Luckily Netbeans has such a formatting option so I can keep to our existing format. But you do bring up a good point that there have been some formatting rules I've been breaking :)

Should this formatting rule be added to the formatting standard if it isn't already there?